mlrun/db/httpdb.py

```diff
@@ -1509,7 +1531,9 @@ def list_entities(
         return resp.json()["entities"]
 
     @staticmethod
-    def _generate_partition_by_params(partition_by, rows_per_partition, sort_by, order):
+    def _generate_partition_by_params(
+        partition_by_cls, partition_by, rows_per_partition, sort_by, order
```

Comment:
You added this parameter but are not using it. Looks like you forgot to implement this method.